crates/stages/src/test_utils/macros.rs

```diff
@@ -142,4 +111,42 @@ macro_rules! stage_test_suite {
     };
 }
 
+macro_rules! stage_test_suite_ext {
```

Comment:
docs on this one? how it differs from e.g. `stage_test_suite!`

Reply:
added